app/actions/session.jsx

```diff
@@ -0,0 +1,17 @@
+import Alt from 'alt_flux';
+import sessionSource from 'sources/session';
+
+export default Alt.createActions(class SessionActions {
+  create(user) {
+    return (dispatch) => {
+      sessionSource.create(user).then(result => dispatch(result));
+    };
+  }
+
+  destroy(user) {
+    return (dispatch) => {
+      sessionSource.delete(user);
+      dispatch(user);
+    };
+  }
+});
```

Comment:
or event better, probably we could use decorating approach via [alt-utls](https://github.com/altjs/utils/blob/master/src/decorators.js#L46):